src/System.Text.Json/tests/Serialization/TestClasses.cs

```diff
@@ -32,6 +32,41 @@ public enum SampleUInt64Enum : ulong
         Max = ulong.MaxValue
     }
 
+    public struct SampleStruct
```

Comment:
Same here. I think this should be SimpleStruct (to follow the naming scheme for other classes/tests here).

I believe the recent enums added to the test are the only outliers.